lib/api/openid_connect/token_endpoint.rb

```diff
@@ -0,0 +1,65 @@
+module Api
+  module OpenidConnect
+    class TokenEndpoint
+      attr_accessor :app
+      delegate :call, to: :app
+
+      def initialize
+        @app = Rack::OAuth2::Server::Token.new do |req, res|
+          o_auth_app = retrieve_client(req)
+          if app_valid?(o_auth_app, req)
+            handle_flows(o_auth_app, req, res)
+          else
+            req.invalid_client!
+          end
+        end
+      end
+
+      def handle_flows(o_auth_app, req, res)
```

Comment:
Unused method argument - `o_auth_app`. If it's necessary, use `_` or `_o_auth_app` as an argument name to indicate that it won't be used.<br>Assignment Branch Condition size for handle_flows is too high. [17.58/15]